test/spec/features/service/scale_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'service scale' do
+  it 'scales a service' do
+    run("kontena service create test-1 redis:3.0")
+    k = kommando("kontena service deploy test-1")
+    expect(k.run).to be_truthy
+    k = kommando("kontena service scale test-1 3")
+    expect(k.run).to be_truthy
+
```

Comment:
Should we verify that three instances are actually created within some decent time?